ladybug/src/main/java/org/frankframework/ibistesttool/Debugger.java

```diff
@@ -241,7 +241,7 @@ public Object storeInSessionKey(String correlationId, String sessionKey, Object
 
 	@Override
 	public <T> T showValue(String correlationId, String label, T value) {
-		return testTool.inputpoint(correlationId, null, label, value);
+		return testTool.outputpoint(correlationId, null, label, value);
```

Comment:
Nu worden alle `showValue` methodes omgedraaid, misschien dat je deze kan hernoemen of een argument in/out toevoegen?

Reply:
I looked at every usage of this function and changed it accordingly.